### DISCUSSION TITLE
[Bug] Article counter isn't displaying correctly when translated in french

### DISCUSSION THREAD

**Discussion by ThbtSprt** [django-wiki/django-wiki#1384] [Bug] [answered]
### Describe the bug

The french version of django-wiki displays the article counter without spaces between the words, as you can see below.

By the way, the translation isn't correct : 

- _"There is 1 article"_ should be translated : _"Il y a 1 article"_ 
- _"There are 2 articles"_ should be translated : _"Il y a 2 articles"_ 

I'm not familiar with Transifex so I don't know how to fix it.

Thank you for your help.

<img width="983" alt="Capture d’écran 2024-12-06 à 22 12 55" src="https://github.com/user-attachments/assets/e430ba19-b553-49c8-88b0-446ea056dacf">

### Steps to reproduce.

1. Set the following django language settings : 
```python
LANGUAGES = [("fr", "French")]
LANGUAGE_CODE = "fr"
```

2. Browse to [http://localhost:8000/_dir/](http://localhost:8000/_dir/)

### Expected behavior

- _"There is 1 article"_ should be translated : _"Il y a 1 article"_ 
- _"There are 2 articles"_ should be translated : _"Il y a 2 articles"_ 

### Python Version

3.11

### Django Version

4.2

**Comment by benjaoming** [comment, marked as answer]
Hey! Translations are handled via Transifex. If you sign up, you will find the translation string in this search: https://app.transifex.com/django-wiki/django-wiki/translate/#fr/core/150161891?q=text%3A'there+is'

**Comment by ThbtSprt** [comment]
I changed the format on transifex, thank you !
